pom.xml

```diff
@@ -1001,11 +1004,26 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <executions>
           <execution>
-            <id>run-integration-tests</id>
+            <id>run-minicluster-integration-tests</id>
             <goals>
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <skip>${accumulo.skipMiniClusterInFailsafe}</skip>
+              <groups>${accumulo.miniclusterTests}</groups>
+            </configuration>
+          </execution>
+          <execution>
```

Comment:
Yes, they would be sharing a log directory, but aren't the contents going to be unique (e.g. class names are stil unique).

> How does failsafe match up the two different integration-test mojo outputs with their corresponding verify mojo work if both execute?

I.. don't understand the question :)